taxonomies/categories.txt

```diff
@@ -43820,18 +44100,28 @@ de:Putenschnitzel
 fi:kalkkunaleikkeleet
 fr:Escalopes de dinde, escalopes de dindes, escalope de dinde, escalope de dindes
 nl:Kalkoenkoteletten, Kalkoenkotelet
+ciqual_food_code:en:36304
+ciqual_food_name:en:Turkey, escalope, raw
 
 <en:Turkeys
 fr:Rôtis de dinde, rôtis de dindes, rôti de dinde, rôti de dindes
+en:Roasted turkey meat, baked turkey meat
+ciqual_food_code:en:36302
+ciqual_food_name:en:Turkey, meat, roasted/baked
 
 <en:Turkeys
-en:Turkey thighs
+en:Turkey thighs, Turkey leg meat
 fr:Cuisses de dinde, cuisses de dindes, cuisse de dinde, cuisse de dindes
 nl:Kalkoendijen
+ciqual_food_code:en:36305
+ciqual_food_name:en:Turkey, leg, meat and skin, raw
 
 <en:Turkeys
 <fr:Allumettes de volaille
 fr:Allumettes de dinde, alumettes de dinde, allumette de dinde, alumette de dinde
+en:Poultry ham in cube grated or minced
```

Comment:
```suggestion
en:Minced turkey ham, grated turkey ham, diced turkey ham
```

--> what we put in the "en:" name must correspond to the existing translations like fr:

Reply:
if the Ciqual category is more general, that's fine, we can associate it anyway, but we don't make it the canonical name of the category.